plugins/language_marte.lua

```diff
@@ -0,0 +1,42 @@
+-- mod-version:3
+local syntax = require "core.syntax"
+
+syntax.add {
+  name = "MARTe",
+  files = { "%.cfg$", "%.marte$" },
```

Comment:
I'm a bit reluctant about letting this apply to `.cfg`...

Reply:
I agree, I will only use marte extension.